ddtelemetry-ffi/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 # This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present Datadog, Inc.
 
 [package]
-name = "ddtelemetry-ffi"
+name = "datadog-telemetry-ffi"
```

Comment:
If you rename the crate, it'd be a good idea to move it to the same path as it's name